pydicom/tests/test_multival.py

```diff
@@ -129,6 +132,17 @@ def test_not_equal(self):
         self.assertFalse(multival != multival2)
         self.assertTrue(multival != multival3)
 
+    def test_str_rep(self):
+        """MultiValue: test print output"""
+        multival = MultiValue(IS, [])
+        self.assertEqual('', str(multival))
+        multival = MultiValue(compat.text_type, [1, 2, 3])
```

Comment:
Just a style point, but I thought we were moving to bare `assert`s, like
```assert '' == str(multival)```

Reply:
Ah, did that without thinking, as the rest of the test uses the same.

Reply:
Changed now.